.github/workflows/bindings_java.yml

```diff
@@ -185,7 +185,7 @@ jobs:
         run: bash ./scripts/merge_local_staging.sh $LOCAL_STAGING_DIR/staging ~/linux-x86_64-local-staging/staging ~/osx-x86_64-local-staging/staging ~/osx-aarch_64-local-staging/staging # ~/windows-x86_64-local-staging/staging
 
       - name: Deploy local staged artifacts
-        if: ${{ github.event_name != 'pull_request' }}
+        if: false # ASF org-wise token are not allowed to directly push to release repository
```

Comment:
By the way, CI fails at `stage-releases` job.

https://github.com/apache/incubator-opendal/actions/runs/5397276885/jobs/9801810125

Maybe we should disable the entire `stage-releases` job instead

Reply:
You're right. Let me prepare a follow-up